source/dhtslib/coordinates.d

```diff
@@ -256,6 +271,99 @@ struct Coordinates(CoordSystem cs)
         }
     }
 
+    /// Convert coordinate to another based system using shortcuts
+    auto to(T: ZBHO)()
+    {
+        return this.to!(CoordSystem.zbho);
+    }
+
+    /// Convert coordinate to another based system using shortcuts
+    auto to(T: OBHO)()
+    {
+        return this.to!(CoordSystem.obho);
+    }
+
+    /// Convert coordinate to another based system using shortcuts
+    auto to(T: ZBC)()
+    {
+        return this.to!(CoordSystem.zbc);
+    }
+    
+    /// Convert coordinate to another based system using shortcuts
+    auto to(T: OBC)()
+    {
+        return this.to!(CoordSystem.obc);
+    }
+
+    /// make a new coordinate pair with a value of 
+    /// this.start + off and this.end + off
+    auto offset(T)(T off)
+    if(isIntegral!T)
+    {
+        return Coordinates!(cs)(cast(long)(this.start + off), cast(long)(this.end + off));
```

Comment:
I had intended for most operations with `Coordinates` to return a copy. I tend to prefer this but it can easily be changed. Honestly my "copy" logic only makes sense if we treat/make all coordinates as immutable.

Reply:
Offset is both a verb (implying mutable) and a noun (implying a copy. Could we give it a slightly different name?  I won't be hard line on this point, just wondering. 

`withOffset`,  `offseted` (which is in fact a word ;) ), not sure what else. may be more trouble than its worth